src-plugins/itkFilters/itkFiltersProcessBase.cpp

```diff
@@ -89,3 +88,23 @@ void itkFiltersProcessBase::eventCallback ( itk::Object *caller, const itk::Even
 
     source->emitProgress((int) (processObject->GetProgress() * 100));
 }
+
+dtkSmartPointer<medAbstractImageData> itkFiltersProcessBase::getInputData()
+{
+    return d->inputData;
+}
+
+void itkFiltersProcessBase::setInputData(dtkSmartPointer<medAbstractImageData> inputData)
+{
+    d->inputData = inputData;
+}
+
+dtkSmartPointer<medAbstractImageData> itkFiltersProcessBase::getOutputData()
```

Comment:
there is already an `itkFiltersProcessBase::output()` function which does the same

Reply:
Not quite: it returns a raw pointer, not a smartPointer, so for homogeneity I wanted a proper getter.

Reply:
huuum, i don't like an (almost) duplicated function, but we don't have too much time